spec/support/shared_controller_examples.rb

```diff
@@ -0,0 +1,125 @@
+# HTTP status code examples ----------------------------------------------------
+
```

Comment:
OoooooOOOOoo shared contexts!!! This is super shiny and lots of folks in the industry don't know about these.